### PR TITLE
Fix missing a reference to the official m2 translations project in gi…

### PR DESCRIPTION
**Pull request by diazwatson**
## Purpose of this pull request

This pull request (PR) adds a reference to the official m2 translations GitHub project https://github.com/magento-l10n

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.2/frontend-dev-guide/translations/xlate.html
- https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/translations/xlate.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.2/frontend-dev-guide/translations/xlate.md [hunk 1]
```diff
@@ -164,6 +164,8 @@ If you need help understanding the context or meaning of a UI string, or have qu
 * [Use translation dictionary to customize strings]
 * [Translate theme strings]
 * [Example theme translation dictionary]
+* [Magento translations GitHub project]
+
```
Comment (ronak2ram) [inline guides/v2.2/frontend-dev-guide/translations/xlate.md]:
```suggestion

```

Comment (ronak2ram) [reply guides/v2.2/frontend-dev-guide/translations/xlate.md]: Hi @diazwatson
Thanks for your contributions. Can you please remove the extra line.

**Comment by diazwatson** [comment]
@ronak2ram  done

**Comment by ronak2ram** [comment]
Hi @diazwatson 
Thanks for the update.
There is also one small changes. Actually, I see there are two extra lines 168,169 for v2.2 so we have to remove line no.168.
https://github.com/magento/devdocs/pull/5126/files#diff-01f43e2a0588f06fe1640588794f5b21R168

**Comment by diazwatson** [comment]
@ronak2ram Done

**Pull request merged**